### ISSUE TITLE
Update historic maven package definitions

### ISSUE THREAD

**Issue by nellshamrell** [clearlydefined/service#799]
Definitions for Maven components in ClearlyDefined used to have incorrect source and other urls (see #755).

With this pull request #794, all new Maven definitions will have the correct urls. This has been merged and deployed.

However, there is still the matter of updating all of the existing Maven definitions (800,000+) to have correct URLs.

I initially took the approach of [creating a script to update the "definitions-paged" Mongo DB](https://github.com/clearlydefined/correct-cd-maven-urls). This worked in my local development environment.

However, it did not work in our [development environment](https://dev.clearlydefined.io/?sort=releaseDate&sortDesc=true)

This is because, in our development and production environments, we use Azure blob storage (along with Mongo) to store definition information.

As @jeffmcaffer put it in our internal Slack:

"the definitions come from blob storage. You could completely delete the cosmos db and ClearlyDefined would continue to work (though search would be affected).
2:12
The key takeaway here is that blob is the golden store. Everything else should be derived from that."

A better approach to update the historic data may be to:

* Modify [this API call](https://github.com/clearlydefined/service/blob/324017ef0ccfe73eb9d1593faf777d19b621a216/routes/definitions.js#L65) to take a parameter to force recomputation (and storage) of the definition. This is currently done automatically for calls to this API from a localhost, but there is not currently a way to do this outside of localhost. This should be a straightforward change.
* Download info for all 800,000 maven components into a big json file
* Write a script to parse through the 800,000 components and call POST /definitions for each of them (sleeping for at least a second or two between each one)
* Likely call the script on chunks of 100,000, rather than trying to do all 800,000 at once.

**Comment by nellshamrell** [comment]
I've updated this repo https://github.com/clearlydefined/correct-cd-maven-urls, which contains a script to correct the records through GET /definitions requests. 

I've tested this in my local environment and in the azure dev environment and it seems to work for the vast majority of records. There are a few that time out that we will likely need to handle individually, but updating the vast majority through an automated scripts will meet our needs at this time.
